services/consensus_topic_info.proto

```diff
@@ -31,60 +38,121 @@ import "duration.proto";
 import "timestamp.proto";
 
 /**
- * Current state of a topic.
+ * A query response describing the current state of a topic for the Hedera
+ * Consensus Service (HCS).
  */
 message ConsensusTopicInfo {
     /**
-     * The memo associated with the topic (UTF-8 encoding max 100 bytes)
+     * A short description of this topic.
+     * <p>
+     * This value, if set, MUST NOT exceed 100 bytes when encoded as UTF-8.
      */
     string memo = 1;
 
-
     /**
-     * When a topic is created, its running hash is initialized to 48 bytes of binary zeros.
-     * For each submitted message, the topic's running hash is then updated to the output
-     * of a particular SHA-384 digest whose input data include the previous running hash.
-     * 
-     * See the TransactionReceipt.proto documentation for an exact description of the
-     * data included in the SHA-384 digest used for the update.
+     * The latest running hash of the topic.
+     * <p>
+     * This 48-byte field is the output of a SHA-384 digest with input
+     * data determined by the current version of the running hash algorithm
+     * used by the network.<br/>
+     * All new transactions SHALL use algorithm version `3`.<br/>
+     * The bytes of each uint64 or uint32 encoded for the hash input
+     * MUST be in Big-Endian format.
+     * <p>
+     * <hr/>
```

Comment:
Just wondering in general on the guidance of when is it advisable or appropriate to add <hr/> tags ?

Reply:
In large blocks of documentation, it is appropriate to add `hr` tags to set apart large groupings of closely related items, or provide visual separation for multiple broad topics.
In general, such usage should be rare, however, as we would prefer not to need such large blocks of documentation in most situations.

Reply:
Updated.